src/util/jsPlumbHelpers.ts

```diff
@@ -25,8 +38,8 @@ export const connectLinkItems = async (source, connectionArray, parent, subject,
           [
             'Custom',
             {
-              create: component => {
-                return container.starButton.current;
+              create: (component: any) => {
```

Comment:
"'component' is declared but its value is never read.", så kan fjernes? 
`create: (component: any) => { `--> `create: () => {`